docs: mention docusaurus.community site in resources (#8943)
facebook/docusaurus: website/community/2-community-docs.mdx

> ---
> title: Community Docs
> hide_title: true
> sidebar_label: Community Documentation
> ---
> 
> ## Community Documentation
> 
> (extends the [Official Docs](/docs))
> 
> The **[Docusaurus.community](https://docusaurus.community)** site can be used for extending the official documentation with more complex examples and full articles on best practices that provide solutions to problems the community often faces.
> 
> (It is also a full React application just like our main site. So customization and experimentation examples are perfectly suited to be hosted there.)
> 
> :::info We might allow direct links to be added in our official documentation that point to the community documentation, on a case by case basis, and as the community requests that such links to articles are useful.
> 
> We are also contemplating further integration into the official documentation at some point in the future. :::

Reviewer: ```suggestion
# Community Documentation

The **[Docusaurus.community](https://docusaurus.community)** site extends the [official docs](/docs) with more complex examples and full articles that recommend best practices and provide solutions to common problems.

It is also a full Docusaurus application just like our main site, so customization and experimentation examples are perfectly suited to be hosted there.

:::info

We might allow direct links to be added in our official documentation that point to the community documentation, on a case-by-case basis, if the community finds such links to articles useful.

We are also contemplating further integration into the official documentation at some point in the future.

:::
```

Not sure why it needs a single H2 heading.
